corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -287,6 +287,19 @@ def _test_user_data_profile_unknown(self, is_web_upload=False):
         )['messages']['rows']
         self.assertEqual(rows[0]['flag'], "Profile 'not_a_real_profile' does not exist")
 
+    def _test_user_data_profile_removal(self, is_web_upload=False):
+        import_users_and_groups(
+            self.domain.name,
+            [self._get_spec(user_profile='')],
```

Comment:
Shouldn't this test operate on a user that already has a profile?  I think this creates a new user (though I'm not certain how these tests work).

Might also be useful to check `user.get_user_data(domain).profile`, which looks up the profile corresponding to the ID.  It looks like it's wrapped in a `if self.profile_id`, so I don't think it'll fail, just seems good to verify.

Reply:
I think Ethan's comment still applies - that the editable user should have an existing profile. Otherwise I think this test would pass even without your changes?